artman/config/loader.py

```diff
@@ -32,6 +34,7 @@
 # TODO(ethanbao): Add a reference link once Artman schema spec is publicly
 # available.
 INVALID_CONFIG_ERROR_MESSAGE_FORMAT = 'Artman YAML %s is invalid.'
+INVALID_USER_CONFIG_ERROR_MESSAGE_FORMAT = 'Artman YAML %s is invalid.'
```

Comment:
Is this repeated error message intentional? We should have different names for the Artman config for an API and the Artman user config.